.github/workflows/release.yml

```diff
@@ -13,6 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
```

Comment:
Could you bump all instances of `checkout` to `v4`?

Reply:
```suggestion
      - uses: actions/checkout@v4
```